cmd/xamarin.go

```diff
@@ -87,13 +88,25 @@ func scanXamarinProject(cmd *cobra.Command, args []string) error {
 	// Xamarin Solution Path
 	xamarinCmd.SolutionFilePath = paramXamarinSolutionFilePath
 	if xamarinCmd.SolutionFilePath == "" {
-		askText := `Please drag-and-drop your Xamarin Solution (` + colorstring.Green(".sln") + `) file,
-and then hit Enter`
+
 		fmt.Println()
-		projpth, err := goinp.AskForPath(askText)
+		log.Infof("Scan the directory for project files")
+		projpth, err := scanForProjectFiles([]ProjectType{xamarinSolution})
```

Comment:
what if we find 1 solution, but unfortunately not the one the user wants to use? the logic does not fail, so the user has no option to use another solution file?

What if:
- return a list of solutions and let the user choose
- setup the tool in the way that solution can be defined via flag/argument
- if flag/arg passed use that solution
- otherwise, scan for solution let the user choose in case of multiple solution files and also print a warning message, that specific solution file can be defined via flag/arg (print the copy-paste ready command to define the solution)